### ISSUE TITLE
L_train vs L_dev naming convention

### ISSUE THREAD

**Issue by daniter-cu** [HazyResearch/snorkel#431]
In tutorials/disease_tagging/Disease_Tagging_Tutorial_4.ipynb you define L_train as the matrix of labeling function labels on all training data. Then later in the same notebook you define L_dev as the ground truth on the dev data. It's confusing.

**Comment by ajratner** [comment]
Agreed! If you want to push a fix that's great otherwise we'll take care of
